pkg/provisioner/ironic/ironic.go

```diff
@@ -512,7 +512,7 @@ func (p *ironicProvisioner) changeNodeProvisionState(ironicNode *nodes.Node, opt
 // details of devices discovered on the hardware. It may be called
 // multiple times, and should return true for its dirty flag until the
 // inspection is completed.
-func (p *ironicProvisioner) InspectHardware() (result provisioner.Result, details *metal3v1alpha1.HardwareDetails, err error) {
+func (p *ironicProvisioner) InspectHardware(inspect bool) (result provisioner.Result, details *metal3v1alpha1.HardwareDetails, err error) {
```

Comment:
Can we give this parameter a more meaningful name like `forceNew`?

Reply:
Fixed, thanks